cpp/ql/test/query-tests/Best Practices/SloppyGlobal/main.cpp

```diff
@@ -5,3 +5,19 @@ int ys[1000000]; // BAD: too short
 int descriptive_name; // GOOD: sufficient
 
 static int z; // GOOD: not a global
+
+int v1;
+int v2;
+template <typename T>
+T v3;
+template <typename T>
+T v4;
+template <typename T>
+T v5;
```

Comment:
Can you annotate these with `GOOD` or `BAD` (no need to also add an explanation).